lib/config.py

```diff
@@ -62,15 +62,19 @@
         dict(help='Branch of the release notes repository to checkout'),
 }
 PUSH_REPO_ARGS = {
+    ('--commit-updates',):
+        dict(help='Commit file updates to local repository'),
+    ('--push-updates',):
+        dict(help='Push file updates to remote repository'),
     ('--push-repo-url',):
         dict(help='URL of the repository used for pushing'),
     ('--push-repo-branch',):
         dict(help='Branch of the repository used for pushing',
              default='master'),
-    ('--committer-name',):
-        dict(help='Name used when creating a commit and bumping spec files'),
-    ('--committer-email',):
-        dict(help='Email used when creating a commit and bumping spec files'),
+    ('--updater-name',):
+        dict(help='Name used when updating files'),
```

Comment:
I think `updating spec files and committing changes` would be clearer.
Or maybe `Name used to update spec file changelog and create git commit`